### PR TITLE
Closes #120: Fix crash with unknown decodedValue

### LINKED ISSUE #120: Investigate 500
https://safe-client-rinkeby.staging.gnosisdev.com/v1/safes/0x441E604Ad49602c0B9C0B08D0781eCF96740786a/transactions

```json
{
  "transactionServiceBaseUrl":"https://safe-transaction.staging.gnosisdev.com/api/v1",
  "name":"safe-client-gateway",
  "version":"0.2.0-24-g3e3d8de",
  "buildNumber":"142"
}
```

### PR DESCRIPTION
**Pull request by rmeissner**
Closes #120

### CODE REVIEW

**Comment by jpalvarezl** [comment]
Somehow I can't request your review @rmeissner on your own PR 😓 

There is a lot of `serde` magic that @nlordell showed me :D 

I also sneaked in the field rename for confirmations.

File: src/utils/mod.rs [hunk 1]
```diff
@@ -6,6 +6,7 @@ use rocket::http::uri::Absolute;
 pub mod cors;
 pub mod context;
 pub mod cache;
+pub mod json;
```
Comment (rmeissner) [inline src/utils/mod.rs:9]: Why is this required?

File: src/utils/json.rs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Deserializer};
+
+pub fn try_deserialize<'de, D, T>(deserializer: D) -> Result<Option<T>, D::Error>
```
Comment (rmeissner) [inline src/utils/json.rs:3]: This is automagically applies for all options? Or when is this registered to be used by serde?

Comment (rmeissner) [reply src/utils/json.rs:3]: Ahh I see serialize_with ;)

File: src/utils/mod.rs (hunk 1 above)
Comment (jpalvarezl) [reply src/utils/mod.rs:9]: Added the `json` utils. At first we had an `Unknown` struct wrapping `serde::Value` to dump all the json, but that proved hard to test (the inner structure was hard to determine from outside)

**Review by jpalvarezl — APPROVED** [review]
Proxy approve from @rmeissner

File: src/utils/mod.rs (hunk 1 above)
Comment (jpalvarezl) [reply src/utils/mod.rs:9]: This could be a good place for custom (de)serializers for generic types

**Pull request merged**